### ISSUE TITLE
Installation problem node v4.1.2, Mac OS 10.8.5

### ISSUE THREAD

**Issue by antonharald** [mikeseven/node-webgl#42]
Hi! After installing all necessary dependencies, I tried:

sudo npm install node-webgl

But I keep gettings this:

../src/common.h:15:24: fatal error: GLFW/glfw3.h: No such file or directory

But:

ls /usr/local/include/GLFW/
glfw3.h		glfw3native.h	npm-debug.log

Any ideas, what could go wrong in here? Somehow the node-gyp cannot find the .h file even though it is there. Using node-gyp directly inside the repo clone gives me another error:

../node_modules/nan/nan.h:324:27: error: redefinition of 'template<class T> v8::Local<T> Nan::imp::NanEnsureHandleOrPersistent(const v8::Local<T>&)'

Which confuses me even more. Any help would be appreiciated a lot!

**Comment by mikeseven** [comment]
I haven't moved node-webgl to node 4 yet. But I started with glfw. Gotta find some time to finish it asap!



On Fri, Oct 16, 2015 at 6:26 AM -0700, "antonharald" <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi! After installing all necessary dependencies, I tried:

sudo npm install node-webgl

But I keep gettings this:

../src/common.h:15:24: fatal error: GLFW/glfw3.h: No such file or directory

But:

ls /usr/local/include/GLFW/
glfw3.h glfw3native.h npm-debug.log

Any ideas, what could go wrong in here? Somehow the node-gyp cannot find the .h file even though it is there. Using node-gyp directly inside the repo clone gives me another error:

../node_modules/nan/nan.h:324:27: error: redefinition of 'template v8::Local Nan:[:imp:]:NanEnsureHandleOrPersistent(const v8::Local&)'

Which confuses me even more. Any help would be appreiciated a lot!

-
